vst.go

```diff
@@ -758,6 +763,27 @@ type (
 	}
 )
 
+// GetVal should be called in ProcessDoubleFunc or ProcessFloatFunc and will be called in GetDisplayVal.
+// It returns the plain Value or the return value of GetValue, when it was set for the Parameter
+func (e Parameter) GetVal() float32 {
```

Comment:
Rename to `GetValue`?

Reply:
Then there will be a conflict, i changed name of the function inside the struct `GetValueFunc`.